lib/commands/element.js

```diff
@@ -351,8 +352,9 @@ export async function setValue (text, elId) {
 
 export async function performActionWithKeyboard (action, elId) {
   const kbSelector = '//StdDlgKeyboardItem[@focused="true"] | ' +
-                     '//Keyboard[@focused="true"] | ' +
-                     '//CustomKeyboard';
+    '//Keyboard[@focused="true"] | ' +
+    '//CustomKeyboard | ' +
+    '//BetaCustomKeyboard';
```

Comment:
interesting. Do they have various keyboard kind..?

Reply:
Yeah. I think a long term solution will be to allow the user to define what counts as a keyboard